Results.html

```diff
@@ -21,14 +22,14 @@
     <div class = "nav">
         <ul>
             <li> <a href="index.html"> About </a> </li>
-            <li> <a href="Results.html" class="selected"> Results </a> </li>
+            <li> <a href="results.html" class="selected"> Results </a> </li>
             <li> <a href="map.html"> Find the Coffee </a></li>
         </ul>
     </div>
 </header>
 <section class="roaster-winner">
```

Comment:
I don't love the class "roaster winner" because I think we should move to a more generic term.  So that the same table could eventually be used for other things (non coffee related).   Something like "banner item" or something that could be reused.   Eventually we will be pulling the winner from the JSON files and the HTML won't know who won.